modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -99,7 +132,7 @@ When the pods are created for the workload object, the VPA constantly monitors t
 
 [NOTE]
 ====
-The workload object must specify a minimum of two replicas in order for the VPA to monitor and update the pods. If the workload object specifies one replica, the VPA does not delete the pod to prevent application downtime. You can manually delete the pod to use the recommended resources.
+By default, workload objects must specify a minimum of two replicas in order for the VPA to automatically delete their pods. Workload objects that specify fewer replicas than this minimum are not deleted. If you manually delete these pods, when the workload object redeploys the pods, the VPA does update the new pod with its recommendations. You can change this minimum by modifying the `VerticalPodAutoscalerController` object as shown shown in _About changing the VPA minimum value_.
```

Comment:
Noticed that this note has singular "does update the new pod" whereas the other two instances had plural "pods". Dunno if intentional tho.

Reply:
Last-minute change to plural pods. Thanks for catching the one I missed!